cmd/internal/cli/startvm_darwin.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cli
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"os/user"
+	"path/filepath"
+	"runtime"
+	"strconv"
+	"strings"
+	"syscall"
+
+	"github.com/sylabs/singularity/internal/pkg/buildcfg"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+)
+
+func startVM(sifImage, singAction, cliExtra string, isInternal bool) error {
+	// Setup some needed variables
+	hdString := fmt.Sprintf("2:0,ahci-hd,%s", sifImage)
+	bzImage := fmt.Sprintf(buildcfg.LIBEXECDIR+"%s"+runtime.GOARCH, "/singularity/vm/syos-kernel-")
+	initramfs := fmt.Sprintf(buildcfg.LIBEXECDIR+"%s"+runtime.GOARCH+".gz", "/singularity/vm/initramfs_")
+
+	// Default xhyve Arguments
+	defArgs := []string{"-A", "-m", VMRAM, "-c", VMCPU, "-s", "0:0,hostbridge", "-s", hdString, "-s", "31,lpc", "-l", "com1,stdio"}
+
+	if len(BindPaths) > 8 {
+		sylog.Fatalf("Maximum of 8 bind mounts")
+	}
+
+	// Bind mounts
+	singBinds := []string{""}
+
+	slot := 5
```

Comment:
I can't find documentation for this. Is there something special about PCI slot 5 on xhyve?

Reply:
No. I just chose `5` as the starting slot for the bind points